src/app/containers/OnDemandImage/index.jsx

```diff
@@ -15,7 +14,7 @@ const paddingDir = ({ dir }) => `padding-${dir === 'rtl' ? 'left' : 'right'}`;
 const ImageContainer = styled.div`
   padding: ${GEL_SPACING_QUAD} 0 ${GEL_SPACING_TRPL};
   ${paddingDir}: ${GEL_SPACING_DBL};
-  @media (max-width: ${GEL_GROUP_1_SCREEN_WIDTH_MAX}) {
+  @media (max-width: 22.5rem) {
```

Comment:
`22.5rem` is `360px`. By using `max-width: 22.5rem`, the image will not appear at 360px. The wording of the issue is slightly confusing because the title says below 360 but the description says above 360. But based on the designs, the image needs to show at `360px` inclusive. This means the `max-width` here should be 359/16 which is `22.4375rem`.